src/app/static/src/app/components/header/DataExplorationSupportMenu.vue

```diff
@@ -3,8 +3,14 @@
         <drop-down text="support" :right="true" :delay="true" style="flex: none">
             <a class="dropdown-item"
                @click="toggleErrorReportModal"
+               tabindex="0"
```

Comment:
Noticed this item wasn't styled consistently with other buttons in the drop down. This shows as black text. Setting `tabindex` fixes this, copied over from main page support menu.